build.gradle

```diff
@@ -238,7 +238,7 @@ task iOSImages {
 """
             def render = new File(imageset, basename + ".png")
             exec {
-                commandLine System.getenv('INKSCAPE') ?: "inkscape", "-z", "-e", render, "-w", "380", details.file.path
+                commandLine System.getenv('INKSCAPE') ?: "inkscape", "-o", render, "-w", "380", details.file.path
```

Comment:
Why do you change options? Also you use png on both Android and iOS. So again this is out of scope of this PR

Reply:
On anything newer than Inkscape 0.92.x those options no longer work so I had to change them to get this gradle task to run. I'll revert it for now and make it a separate PR.